src/networks/local_sandwich.jl

```diff
@@ -104,3 +71,148 @@ function virtualspace(O::PEPOSandwich, dir)
         virtualspace(bra(O), dir)',
     ])
 end
+
+## Only PEPO layers
+# In a CTMRG contraction, the top physical leg of the top PEPOTensor is contracted with the bottom physical leg of the bottom PEPOTensor
+
+const PEPOLayersSandwich{N,P<:PEPOTensor} = Tuple{Vararg{P,N}}
```

Comment:
I think I prefer `PEPOTrace` as a name here (and then everywhere else correspondingly). It's not really a sandwich, since there's no well-defined slices on either side.

Reply:
Or at the very least `PEPOTraceSandwich`, I think the `Layers` does not add much clarity.